.github/workflows/portal-loop.yml

```diff
@@ -46,3 +48,57 @@ jobs:
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+  test-portal-loop-docker-compose:
```

Comment:
Dumb question, but do we need to close the containers somehow in the CI?

Reply:
Nop, github actions garbage collect the docker engine